### ISSUE TITLE
Listener state: Not listening

### ISSUE THREAD

**Issue by gmbersales** [stascorp/rdpwrap#1224]
![image](https://user-images.githubusercontent.com/2199656/100924096-655c7500-349d-11eb-9b59-8d7bea845131.png)
![image](https://user-images.githubusercontent.com/2199656/100924396-cb48fc80-349d-11eb-8bc9-c64a0d5fdaeb.png)

Windows 10 build 19042.662
I restarted TermServ via RDPWInst.exe -r and got StartService error (code 1056)

rdpwrap.ini might not be current. 17763.1

**Comment by eslamiar** [comment]
Was running fine up until updates last night. Believe it’s 20H2 related, removed most recent updates from Windows rebooted and working again.

**Comment by strarsis** [comment]
@eslamiar: Correct! Uninstalling the patch `KB4586853` (and rebooting, reinstalling RDPWrap) fixed the issue for me.
Related: https://github.com/stascorp/rdpwrap/issues/1223

**Comment by eslamiar** [comment]
Is there new .ini yet to resolve with latest updates ?

**Comment by MabeyC** [comment]
I have the same error message. Following...

**Comment by eslamiar** [comment]
The auto-update tool works here, tested on (1) machine thus far with latest updates. 

https://github.com/asmtron/rdpwrap/blob/master/binary-download.md

**Comment by NeverMorewd** [comment]
I have the same error message. Following...
